web/templates/buddy_list/section_header.hbs

```diff
@@ -1,4 +1,4 @@
-<h5 id="{{id}}" class="buddy-list-heading no-style tippy-zulip-tooltip hidden-for-spectators" data-tippy-content="{{tooltip_text}}" data-tippy-placement="bottom">
+<h5 id="{{id}}" class="buddy-list-heading no-style hidden-for-spectators">
```

Comment:
I took the tooltip out of here and moved to the logic above because 

(1) left-alignment wasn't working because the tooltip was attaching to a nearby html element instead of `body` and didn't have space 

(2) I wanted to add the delay

But maybe there's a simpler way of making this change?

Reply:
Hmmmm. I'm not sure I'm following your comment here based on the code in this area. It might be worth studying how delays and tooltips are set on different headings in the left side bar, but I don't trust that that's the help you're looking for.

Reply:
Essentially there's a way of doing tooltips with `tippy-zulip-tooltip` and `data-tippy-content`, but it didn't seem to support the two things I needed above, so I switched to the code in `buddy_list.initialize_tooltips`. I think the left sidebar code isn't significantly different from what I ended up with here, so I'll leave it as is for now and let @timabbott share if he wants anything different from this commit.